src/loaders/svg/tvgSvgPath.cpp

```diff
@@ -435,7 +437,8 @@ static void _processCommand(vector<PathCommand>* cmds, vector<Point>* pts, char
         case 'a':
         case 'A': {
             _pathAppendArcTo(cmds, pts, cur, curCtl, arr[5], arr[6], arr[0], arr[1], arr[2], arr[3], arr[4]);
-            *cur = {arr[5] ,arr[6]};
+            *cur = *curCtl = {arr[5] ,arr[6]};
```

Comment:
I think this fix is a different issue than the parsing issue. so I recommend splitting up with a new PR or new commit.

Reply:
done.
I'll change this commit after we will agree how it should look like.